app/views/shared/_footer.html.erb

```diff
@@ -1,14 +1,16 @@
-<footer>
-  <div class="container-fluid">
-    <div class="row">
-      <div class="col-md-7">&copy; <%= link_to(t('mezuro_team'), '/humans.txt') %> 2013-2016</div>
-      <div class="col-md-5">
-        <div class="container-fluid">
-          <div class="row">
-            <div class="col-md-2"><%= link_to(image_tag('agplv3-88x31.png'), 'http://www.gnu.org/licenses/agpl-3.0-standalone.html') %></div>
-            <div class="col-md-2"><%= link_to image_tag('fork-me.png'), 'https://github.com/mezuro/prezento' %></div>
-            <div class="col-md-2"><%= image_tag 'usp-cloud-nuvem-logo.png' %></div>
-            <div class="col-md-6"><%= link_to image_tag('banner-ccsl.png', height: '40', width: '190'), 'http://ccsl.ime.usp.br/' %></div>
+<footer class="footer">
+  <div class="container">
+    <div class="container-fluid">
+      <div class="row">
+        <div class="col-md-7">&copy; <%= link_to(t('mezuro_team'), '/humans.txt') %> 2013-2016</div>
+        <div class="col-md-5">
+          <div class="container-fluid">
+            <div class="row">
+              <div class="col-md-3"><%= link_to(image_tag('agplv3-88x31.png'), 'http://www.gnu.org/licenses/agpl-3.0-standalone.html') %></div>
+              <div class="col-md-3"><%= link_to image_tag('fork-me.png'), 'https://github.com/mezuro/prezento' %></div>
+              <div class="col-md-3"><%= image_tag 'usp-cloud-nuvem-logo.png' %></div>
```

Comment:
Maybe we should take this opportunity to remove this logo (we were kicked out of there).